.travis.yml

```diff
@@ -0,0 +1,26 @@
+# More details on how to configure the Travis build
+# https://docs.travis-ci.com/user/customizing-the-build/
+
+language: java
+
+jdk:
+  - oraclejdk8
+
+#Skipping install step to avoid having Travis run arbitrary './gradlew assemble' task
+# https://docs.travis-ci.com/user/customizing-the-build/#Skipping-the-Installation-Step
+install:
+  - true
+
+#Don't build tags
+branches:
+  except:
+  - /^v\d/
+
+#Build and perform release (if needed)
+script:
+  - ./gradlew build -s
+  # Print output every minute to avoid travis timeout
+  - while sleep 1m; do echo "=====[ $SECONDS seconds still running ]====="; done &
```

Comment:
Did we actually run into timeouts? Is the timeout because there is a process that is not producing output for more than 10 minutes?

Reply:
For the message let us do: "=====[ $SECONDS seconds elapsed -- still running ]====="